explorer/types.go

```diff
@@ -218,6 +217,63 @@ type V2HostAnnouncement struct {
 	chain.V2HostAnnouncement
 }
 
+// V2FileContractResolutionType enumerates the types of file contract resolution.
+type V2FileContractResolutionType interface {
```

Comment:
Seems like we're not using this interface?

Reply:
We need to copy it over from core because of the unexported method

```
impossible type assertion: got.Resolution.(*explorer.V2FileContractRenewal)
        *explorer.V2FileContractRenewal does not implement types.V2FileContractResolutionType (unexported method isV2FileContractResolution
```